### DISCUSSION TITLE
Setting up Zoho SAML with Authentik

### DISCUSSION THREAD

**Discussion by S0ulf3re** [goauthentik/authentik#10662] [Q&A]
I am trying to set up Zoho SAML with Authentik so that I can log into my Zoho account with it. However, whenever I try to login to zoho from the Authentik dashboard. I keep getting this error from Zoho when Zoho's name identifier is set to "email address":

```
The email address in the SAML response is incorrect. Please verify your email address null with your Identity Provider and try again.
```
When I set it to "Unspecified", I get the error:

```
The email address <Authentik user NameID attribute>  cannot be found. Contact your admin to resolve the issue
```

I've made sure to use the metadata files for both Zoho and Authentik. However, I think that Authentik may require some additional parameters. But I do not know what to set them to. I'm worried that I may need to create a property mapping, but I'm not sure what to put there either.
![Screenshot_20240727-141438](https://github.com/user-attachments/assets/0ee7a5da-2482-465e-853e-9df0e8092144)

**Comment by nyxb** [comment]
Is there anything here? I would also be very interested in how to set authenticity for zoho. 

@BeryJu

**Comment by cf-sewe** [comment]
How did you configure the SAML Provider Property Mapping within Authentik?
In especial, did you configure the email mapping?

**Comment by mgob** [comment]
So, we have this working if the login is initiated from Zoho but if it's IdP initiated we get a nameid-format unspecified and Zoho tells us to go pound sand.

```
<saml:NameID Format="urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified">
```

I for the life of me can't figure out why this is happening, Zoho is telling us this is our problem but, I'm suspecting it may be on them.

Any ideas?
